apollo-adminservice/src/main/scripts/startup.sh

```diff
@@ -137,10 +157,26 @@ if [[ -n "$APOLLO_RUN_MODE" ]] && [[ "$APOLLO_RUN_MODE" == "Docker" ]]; then
     exec $javaexe -Dsun.misc.URLClassPath.disableJarChecking=true $JAVA_OPTS -jar $PATH_TO_JAR
 else
     if [[ -f $SERVICE_NAME".jar" ]]; then
-      rm -rf $SERVICE_NAME".jar"
+        rm -rf $SERVICE_NAME".jar"
```

Comment:
we need to move these 3 lines after the existProcessUsePort check, or the $SERVICE_NAME".jar" would be removed when the process is already running.

Reply:
Done.

This bug has been reproduced on my machine.

The remove of $SERVICE_NAME".jar" will cause `./scripts/shutdown.sh` cannot work.